include/simple2d.h

```diff
@@ -130,13 +135,16 @@ typedef struct {
   int direction;
   int axis;
   int value;
+  int64_t finger_id;
+  int64_t touch_id;
```

Comment:
Huh, didn't even realize joystick ID was an SDL type. Guess I must have copied some code somewhere using `int`. Looks like [they use SDL types](https://forums.libsdl.org/viewtopic.php?p=34966) for future proofing, but I think it makes sense as a principle not to expose SDL types to users (the ones we use in S2D types are all for internal use) so using regular types make sense here. Might actually change to `long long` if you're ok with it.